Samples/tests/SampleTests/BayesianPETests.qs

```diff
@@ -5,13 +5,13 @@ namespace Microsoft.Quantum.Tests {
     open Microsoft.Quantum.Canon;
     open Microsoft.Quantum.Diagnostics;
 
-    operation BayesianPEManualTest() : Unit {
+    // operation BayesianPEManualTest() : Unit {
```

Comment:
Not sure if you meant to commit this. I assume you may want to remove this test temporarily until I update the compiler?

Reply:
I did not mean to comment this, it was a temporary debugging thing I forgot to uncomment again. Thanks for catching!